chapters/ch17.md

```diff
@@ -1,33 +1,33 @@
 ->
-Tập thứ hai của tập phát triển, được gọi là **tập phát triển hộp đen**, sẽ chứa 4500 mẫu còn lại. Bạn có thể sử dụng tập phát triển hộp đen để đánh giá các bộ phân loại một cách tự động bằng cách đo tỉ lệ lỗi chúng. Bạn cũng có thể sử dụng tập này để lựa chọn thuật toán hoặc điều chỉnh tham số. Tuy nhiên, bạn nên tránh trực tiếp phân tích thủ công tập này. Chúng ta sử dụng thuật ngữ "hộp đen" bời vì chúng ta chỉ sử dụng tập này để thu về những đánh giá "hộp đen" của bộ phân loại. 
+Tập con thứ hai của tập phát triển, được gọi là **tập phát triển Blackbox**, sẽ chứa 4500 mẫu còn lại. Bạn có thể sử dụng tập phát triển Blackbox để đánh giá các bộ phân loại một cách tự động bằng cách đo tỉ lệ lỗi chúng. Bạn cũng có thể sử dụng tập này để lựa chọn thuật toán hoặc điều chỉnh tham số. Tuy nhiên, bạn nên tránh trực tiếp phân tích thủ công tập này. Chúng ta sử dụng thuật ngữ "Blackbox" bời vì chúng ta chỉ sử dụng tập con này để thu về những đánh giá "Blackbox" của bộ phân loại. 
 
 ![img](../imgs/C17_02.png)
 
 Why do we explicitly separate the dev set into Eyeball and Blackbox dev sets? Since you will gain intuition about the examples in the Eyeball dev set, you will start to overfit the Eyeball dev set faster. If you see the performance on the Eyeball dev set improving much more rapidly than the performance on the Blackbox dev set, you have overfit the Eyeball dev set. In this case, you might need to discard it and find a new Eyeball dev set by moving more examples from the Blackbox dev set into the Eyeball dev set or by acquiring new labeled data.
 ->
-Tại sao chúng ta lại chia tập phát triển thành tập phát triển nhãn cầu và tập phát triển hộp đen? Khi bạn nhận thêm trực giác về những mẫu trong tập phát triển nhãn cầu, bạn sẽ overfit tập phát triển đó nhanh hơn. Khi bạn thấy chất lượng của tập phát triển nhãn cầu đang tăng nhanh hơn nhiều so với tập phát triển hộp đen, bạn đã overfit tập phát triển nhãn cầu. Trong trường hợp này, bạn thể phải loại bỏ tập phát triển này đi, và tìm một tập khác thay thế bằng cách chuyển các mẫu từ tập phát triển hộp đen sang tập phát triển nhãn cầu, hoặc thu về những mẫu được gán nhãn khác. 
+Tại sao chúng ta lại chia tập phát triển một cách riêng biệt thành tập phát triển Eyeball và tập phát triển Blackbox? Khi bạn hiểu rõ hơn về các mẫu trong tập phát triển Eyeball, bạn sẽ overfit tập phát triển đó nhanh hơn. Khi bạn thấy chất lượng của tập phát triển Eyeball đang tăng nhanh hơn nhiều so với tập phát triển Blackbox, bạn đã overfit tập phát triển Eyeball. Trong trường hợp này, bạn thể phải loại bỏ tập Eyeball đi, tìm một tập khác thay thế bằng cách chuyển các mẫu từ tập phát triển Blackbox sang tập phát triển Eyeball, hoặc thu về những mẫu có nhãn mới. 
```

Comment:
>  chất lượng của tập phát triển Eyeball

bản gốc là "performance on the Eyeball dev" = hiệu quả trên tập phát triển Eyeball. 
> bạn thể phải loại bỏ tập Eyeball đi

--> bạn có thể phải loại bỏ tập Eyeball đi